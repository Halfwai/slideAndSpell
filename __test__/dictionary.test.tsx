import React from 'react';
import { render } from '@testing-library/react-native';

import Dictionary from '@/app/dictionary';

afterEach(() => {
    jest.clearAllMocks();  // Clears all Jest mocks
    jest.useRealTimers();  // Resets timers
});

it(`Dictionary Component renders correctly`, () => {
    const { toJSON } = render(
        <Dictionary/>
    );
    expect(toJSON()).toMatchSnapshot();
});