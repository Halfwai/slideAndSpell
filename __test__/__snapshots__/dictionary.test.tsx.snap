// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dictionary Component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#CBE6F7",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "borderRadius": 10,
        "height": "10%",
        "margin": 10,
        "padding": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "fontFamily": "Quicksand",
          },
          {
            "fontSize": 35,
          },
        ]
      }
    />
  </View>
  <Text
    style={
      [
        {
          "fontFamily": "Quicksand",
        },
        undefined,
      ]
    }
  >
    Definitions Loading...
  </Text>
</View>
`;
